### ISSUE TITLE
Command doesn't activate if a side panel has the focus

### ISSUE THREAD

**Issue by Randy-Astle** [tgrosinger/leader-hotkeys-obsidian#23]
**Describe the bug**
Even though the breadcrumb is displayed in the title bar, if a side panel has the focus, the command isn't invoked.

**To Reproduce**
1. Open a note
2. Click on a side panel
3. Attempt to activate a command using the mapped key(s) (The Command Palette functions when a side panel has the focus.)

**Expected behavior**

**Screenshots**
N/A

**Desktop (please complete the following information):**
- OS: Windows 10
- Obsidian Version: v0.14.5
- Leader Hotkeys Version: 0.2.0
- Editor | Vim key bindings enabled

**Additional context**
N/A

**Comment by canelhasmateus** [comment]
Indeed, this is a limitation of the current design ( Last update ) . 
In order to implement the functionality, its necessary to completely bypass the native shortcut handler.
This means that we directly inject a listener on the document and due to the ways javascript event-bubbing works, the native shortcut handler is called before our own.
